src/mixer.erl

```diff
@@ -195,23 +196,23 @@ find_dupe(Fun, Arity, [#mixin{mod=Name, fname=Fun, arity=Arity}|_]) ->
 find_dupe(Fun, Arity, [_|T]) ->
     find_dupe(Fun, Arity, T).
 
-insert_stubs(Mixins, Specs, EOF, Forms) ->
+insert_stubs(Mixins, Specs, EOFAnno, Forms) ->
     F =
-        fun(#mixin{} = Mixin, {CurrEOF, Acc}) ->
+        fun(#mixin{} = Mixin, {CurrEOFAnno, Acc}) ->
             #mixin{mod=Mod, fname=Fun, arity=Arity, alias=Alias} = Mixin,
-            {CurrEOF + 1,
+            {erl_anno:set_line(erl_anno:line(CurrEOFAnno) + 1, CurrEOFAnno),
```

Comment:
This is where we actually change the line in the annotation (previously one `Line` and now potentially `{Line, Col}`).